src/lib/components/Count/Count.js

```diff
@@ -21,11 +21,29 @@ export default class Count extends Component {
     return <Label color={'blue'}>{totalResults}</Label>;
   }
 
+  renderSpanWithMargin = (text, margin) => {
+    const size = '0.5em';
+    let style;
+    switch (margin) {
+      case 'left':
+        style = { marginLeft: size };
+        break;
+      case 'right':
+        style = { marginRight: size };
+        break;
+      default:
+        style = { margin: `0 ${size}` };
+    }
+    return <span style={style}>{text}</span>;
+  };
+
   render() {
-    const { loading, totalResults } = this.props;
+    const { loading, totalResults, prefix, suffix } = this.props;
     return (
       <ShouldRender condition={!loading && totalResults > 0}>
+        {this.renderSpanWithMargin(prefix, 'right')}
```

Comment:
Prefix or Suffix might be not defined, so the extra margin should not appear. I would change both to:
`{prefix && this.renderSpanWithMargin(prefix, 'right')}`

WDYT?